package.json

```diff
@@ -199,6 +200,7 @@
     "bundle:css": "node-sass content-src/main.scss -o data/content",
     "bundle:html": "node ./bin/generate-html.js > data/content/activity-streams.html",
     "postbundle": "du -hs ./data/content/*",
+    "bundlestats": "NODE_ENV=production webpack --json | webpack-bundle-size-analyzer",
```

Comment:
Actually, I was thinking of a [different visual module](https://github.com/th0r/webpack-bundle-analyzer). We could almost add this to Circle-CI and stash it in the build histories for historical reasons. Although not sure if we want to slow down the builds by ~10s.

Here's the current output:

<details>
<summary>`$ npm run bundlestats` (clicko expando)</summary>

```sh
> activity-streams@1.1.6 bundlestats /Users/pdehaan/dev/github/mozilla/activity-stream-2
> NODE_ENV=production webpack --json | webpack-bundle-size-analyzer

react: 617.99 KB (47.6%)
wu: 146.36 KB (11.3%)
moment: 137.5 KB (10.6%)
buffer: 47.47 KB (3.66%)
url: 36.04 KB (2.78%)
  punycode: 14.31 KB (39.7%)
  <self>: 21.74 KB (60.3%)
fbjs: 31.21 KB (2.40%)
tippy-top-sites: 31 KB (2.39%)
fathom-web: 30.47 KB (2.35%)
redux: 22.36 KB (1.72%)
react-redux: 19.37 KB (1.49%)
url-parse: 11.23 KB (0.865%)
lodash.debounce: 10.53 KB (0.811%)
object-path: 6.1 KB (0.470%)
lodash: 5.77 KB (0.445%)
process: 5.17 KB (0.398%)
page-metadata-parser: 4.58 KB (0.353%)
querystring: 4.51 KB (0.347%)
reselect: 3.74 KB (0.288%)
base64-js: 3.4 KB (0.262%)
ieee754: 2.01 KB (0.154%)
object-assign: 1.95 KB (0.150%)
invariant: 1.48 KB (0.114%)
object-sizeof: 1.47 KB (0.113%)
hoist-non-react-statics: 1.35 KB (0.104%)
querystringify: 1.27 KB (0.0979%)
fancy-dedupe: 1.13 KB (0.0869%)
symbol-observable: 1.12 KB (0.0866%)
classnames: 1.08 KB (0.0829%)
requires-port: 753 B (0.0566%)
hex-to-rgb: 568 B (0.0427%)
redux-watch: 557 B (0.0419%)
redux-thunk: 529 B (0.0398%)
webpack: 241 B (0.0181%)
isarray: 132 B (0.00993%)
react-dom: 63 B (0.00474%)
<self>: 108.17 KB (8.33%)
```
</details>